data/maintainers.yaml

```diff
@@ -95,8 +95,7 @@
 -
   name: Chris Hughes
   descr: |
-       Chris is an undergrad student in mathematics at Imperial College
-       London (UK).  He has been formalizing mathematics since 2017.
+       Chris is a PhD student at INRIA the Gallinette team in Nantes (France).  He has been formalizing mathematics since 2017.
```

Comment:
```suggestion
       Chris is a PhD student at INRIA with the Gallinette team in Nantes (France).  He has been formalizing mathematics since 2017.
```
?